src/main/java/de/adesso/anki/battle/protocol/commands/UTurnCommand.java

```diff
@@ -1,15 +1,12 @@
-package com.commands;
-
-import de.adesso.anki.battle.world.bodies.Vehicle;
-
-public class UTurnCommand extends Command{
-	
-	public UTurnCommand(){
-		
-	}
-	
-	
-	public void execute(Vehicle vehicle) {
-		// TODO implement UTurn
-	}
-}
+package de.adesso.anki.battle.protocol.commands;
+
+import de.adesso.anki.battle.world.bodies.Vehicle;
+
+public class UTurnCommand extends Command{
+	
+	@Override
+    public void execute(Vehicle vehicle) {
+		// TODO implement UTurn
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=22&issues=AWKWIFf81B6WA4TguGLB&open=AWKWIFf81B6WA4TguGLB)

[ISSUE_KEY]: AWKWIFf81B6WA4TguGLB